views/pulsar/layouts/base.html.twig

```diff
@@ -58,76 +58,79 @@
     <a class="skip-link" href="#skip-target" tabindex="1">Skip to main content</a>
 
 {% block body %}
-    <div class="container" id="top">
+    <div role="main" class="container" id="top">
         <div class="inner-wrapper">
             {{ nav.primary({ 'data': nav_data|default() }) }}
 
-            <header class="toolbar">
-
-                <button tabindex="2" class="mobile-menu-button t-mobile-menu-button" aria-expanded="false" aria-controls="aria-main-nav" aria-label="Toggle main menu">Menu</button>
-
-                {% block notifications %}{% endblock %}
-
-                <div class="nav-user">
-                {% block user_menu %}
-                {{
-                    html.button_dropdown({
-                        'parent_class': 'dropdown user-menu btn--naked',
-                        'label': 'Paul Stanton',
-                        'tabindex': '5',
-                        'items': [
-                            html.link({
-                                "label": html.icon('user') ~ " My details",
-                                "href": "#"
-                            }),
-                            html.link({
-                                "label": html.icon('key') ~ " Change password",
-                                "href": "#"
-                            }),
-                            html.divider(),
-                            html.link({
-                                "label": html.icon('power-off') ~ " Sign out",
-                                "href": "#"
-                            })
-                        ]
-                    })
-                }}
-                {% endblock %}
-                </div>
-
-                {% include '@pulsar/pulsar/components/nav_help.html.twig' %}
+            {% include '@pulsar/pulsar/components/flash.html.twig' %}
```

Comment:
All these changes in base.html.twig look to be out of sync with the more recent changes in develop.